robolectric-processor/src/test/java/org/robolectric/annotation/processing/RobolectricProcessorTest.java

```diff
@@ -32,6 +33,12 @@ public void roboProcessor_supportsPackageOption() {
   }
   
   @Test
+  public void roboProcessor_supportsShouldInstrumentPackageOption() {
+    ASSERT.that(
```

Comment:
org.thruth0?

shouldn't it be org.truth?

Then its assertThat(subject).someAssertion()